### PR TITLE
Update xmind to 3.7.0

### PR DESCRIPTION
**Pull request by neutric**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/xmind.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'xmind' do
-  version '3.6.51'
-  sha256 '9a7d31d61427dcbef5d224c8dc545335bf8ceee6346d224014b535fe1ad9fdf9'
+  version '3.7.0'
```
Comment (vitorgalvao) [inline Casks/xmind.rb]: Why the discrepancy between this and the URL?

File: Casks/xmind.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  sha256 'c4dc8e3eb47d19557fbe5ac837122768eded1321042b064fc3e25f7a2a66ce86'
 
-  url 'http://www.xmind.net/xmind/downloads/xmind-7.5-update1-macosx.dmg'
+  url 'https://www.xmind.net/xmind/downloads/xmind-8-macosx.dmg'
```
Comment (vitorgalvao) [inline Casks/xmind.rb]: And then change this to `url "https://www.xmind.net/xmind/downloads/xmind-#{version}-macosx.dmg"`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/xmind.rb (hunk 1 above)
Comment (neutric) [inline Casks/xmind.rb]: Other package repos (I checked Arch Linux and Chocolatey) refer to it using the 3.x version scheme and the [previous versions page](http://www.xmind.net/download/previous/) indicates that this dual version thing has been going on for more than a year at least. 

Do you still want me to make the requested changes in light of the above?

Comment (vitorgalvao) [inline Casks/xmind.rb]: Yes, please. We can always revert later if needed.

Comment (neutric) [inline Casks/xmind.rb]: Done.

**Review by vitorgalvao — APPROVED** [review]

File: Casks/xmind.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 cask 'xmind' do
-  version '3.6.51'
-  sha256 '9a7d31d61427dcbef5d224c8dc545335bf8ceee6346d224014b535fe1ad9fdf9'
+  version '8'
+  sha256 'c4dc8e3eb47d19557fbe5ac837122768eded1321042b064fc3e25f7a2a66ce86'
 
-  url 'http://www.xmind.net/xmind/downloads/xmind-7.5-update1-macosx.dmg'
+  url 'https://www.xmind.net/xmind/downloads/xmind-#{version}-macosx.dmg'
```
Comment (vitorgalvao) [inline Casks/xmind.rb]: This needs to use double quotes. I’ll make the change.

Comment (vitorgalvao) [inline Casks/xmind.rb]: This needs to use double quotes for the interpolation to work.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/xmind.rb (hunk 3 above)
Comment (neutric) [inline Casks/xmind.rb]: Whoops, sorry, didn't c&p carefully enough. Thanks for fixing!

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8c29d92cb8b7f7843fe01d84bee4a85d96b357a0. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**